src/celengine/render.cpp

```diff
@@ -633,6 +633,16 @@ unsigned int Renderer::getResolution() const
     return textureResolution;
 }
 
+void Renderer::setBoundless(bool b)
```

Comment:
It's better to call this `disableSelectionPoiner()` as I suppose it's the only purpose of this method.